google-cloud-clients/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/OptionAllowTrailingSlash.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 Google LLC
```

Comment:
small nits can you please update Copyright year ?

Reply:
Sure thing!

Reply:
I think this was reversed - the new file should have the copyright year that it was introduced. So this one should be 2019 and the other one should stay 2016.